main.go

```diff
@@ -547,7 +548,8 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
 				if exportMethod == exportoptions.MethodAppStore {
 					iCloudContainerEnvironment = "Production"
 				} else if cfg.ICloudContainerEnvironment == "" {
-					fail("project uses CloudKit, but iCloud container environment input not specified")
+					fail("Your project uses CloudKit and \"iCloud container environment\" input not specified.\n"+
+						"Export method is: %s (For app-store export method Production container environment is implied.)", exportMethod)
```

Comment:
typo `implied`

Reply:
I would log:

Your project uses CloudKit **but** \"iCloud container environment\" input not specified.
The export method is: %s (For app-store export method Production container environment is automatically set.)